tutorials/tour/basics.md

```diff
@@ -90,33 +91,33 @@ println({
 
 ## Functions
 
-Functions are expressions with parameters.
+Functions are expressions that take parameters and compute a result.
```

Comment:
Functions *have* parameters and *take* arguments. It probably doesn't matter, but if you're keeping a running list of things that bug me there you go.